### ISSUE TITLE
Option to disable or configure hud messages like: "Action prevented by Placement Restriction mode"

### ISSUE THREAD

**Issue by dk88711** [maruohon/litematica#76]
Having an option to disable it or move it to corner, text font etc

**Comment by maruohon** [comment]
For the Easy Place and Placement Restriction modes specifically, there is already an option in the 1.12 code to select how or if that message appears (added in 19d39c54dc5).

And on the other hand I think at least the 1.14 versions have some kind of a bug that causes that message to be displayed even when it's not supposed to.

I think I'll also add an option to customize the message overlay position and size, but that option will be in malilib configs because that message renderer is in malilib.

**Comment by KevinDenys** [comment]
Is there an option to turn it off yet?

**Comment by maruohon** [comment]
Still only in the 1.12 version.

I've been working on some somewhat major changes and improvements to certain underlying systems in the mod recently, and after I finish those I'll do the next merge from 1.12 to the 1.13+ versions.

**Comment by KevinDenys** [comment]
Sounds amazing, thanks maruohon. Also is there a way to move the hud messages like offscreen (changing config files?)

**Comment by maruohon** [comment]
Why would you want to move them offscreen? There are some options in the Info Overlays config category for customizing different HUDs, but I guess I could add some more to allow completely disabling different types as well. Although then I'm worried that users will accidentally disable stuff at some point and then they are confused why something doesn't show up anymore.

**Comment by maruohon** [comment]
If you meant that Placement Restriction message in particular, I believe that isn't currently configurable, as it comes via malilib. And now that I think about it, maybe I should change that system a bit so that mods can have their own message renderer instances which can then be configured per-mod or per message renderer type, instead of the mods adding the messages to the one common message renderer in malilib.

**Comment by KevinDenys** [comment]
I was just thinking to move them offscreen for the moment, but like you said the messages are from malilib and not changable. Currently playing on GUI 1 on 4k screen solves the issue from blocking my vision

**Comment by KevinDenys** [comment]
![image](https://user-images.githubusercontent.com/17174443/72209523-91e75700-34af-11ea-94eb-62a38285d2bf.png)

**Comment by maruohon** [comment]
Is this on a server (Spigot/Paper etc.) where the Easy Place mode doesn't work, and using a fast clicker? Otherwise the Easy Place mode with the `easyPlaceHoldEnabled` option enabled and the activation key set to `NONE` with "Allow Empty Keybind" would/should work for less message-spammy way of placing blocks while just holding down the mouse button.

**Comment by KevinDenys** [comment]
Yes, Spigot server easy place not working. And yeah using auto hotkey to help place blocks faster

**Comment by victordlp8** [comment]
I have this same problem in 1.16.1 and I dont know what to do

Can someone help me plz?

**Comment by maruohon** [comment]
There is still no fix other than to go and edit the code. The option to disable this message is still only in the 1.12.2 version. The Easy Place code itself was also rewritten back in March, which also reduces the number of these messages significantly (basically the older version spammed the warning unnecessarily often in some cases).

**Comment by antoine50150** [comment]
Any news for the problem ?...

**Comment by Lardbutt200** [comment]
Are we able to disable the action prevented by easy place mode message yet? Currently using 1.18.1 version

**Comment by maruohon** [comment]
The latest release 1.18.1-0.10.0 now has an option to disable this message, or move it to the action bar: Generic -> `placementRestrictionWarn`.

**Comment by radzio36** [comment]
Mam mały problem, bo jak próbuje użyć funkcji easy place mod to cały czas wyskakuje mi komunikat "Action prevented by easy place mode" i nie wiem, co zrobić nie działa mi to tylko na jednym serwerze, już dłuższy czas próbuję ten problem rozwiązać, ale coś mi nie idzie.
![Zrzut ekranu 2023-11-04 201636](https://github.com/maruohon/litematica/assets/149325261/cae71fcc-35bd-4de2-9e81-22f7bef5956a)
